tools/client-plugins/browser-scripts/typescript-worker.ts

```diff
@@ -46,11 +46,11 @@ let compilerHost: CompilerHost | null = null;
 let cachedVersion: string | null = null;
 
 // NOTE: vfs.globals must only be imported once, otherwise it will throw.
-importScripts('https://unpkg.com/@typescript/vfs@1.6.0/dist/vfs.globals.js');
+importScripts('https://cdn.jsdelivr.net/npm/@typescript/vfs@1.6.0/dist/vfs.globals.js');
 
 function importTS(version: string) {
   if (cachedVersion == version) return;
-  importScripts('https://unpkg.com/typescript@' + version);
+  importScripts('https://cdn.jsdelivr.net/npm/typescript@' + version);
```

Comment:
I don't find alternatives for these two URLs on cdnjs.

Reply:
> For `typescript`, I think it's here: https://cdnjs.com/libraries/typescript.

Here is the thing, we didn't specify the explicit version (currently it is hardcoded as `5`). cdnjs doesn't support this.

Reply:
That's my fault: I should have pinned the full version when I set this up. Let's go with the latest version on cdnjs (5.7.3) so we can at least get off unpkg.

Reply:
Move `typescript` to cdnjs in [`0834eb2` (#59291)](https://github.com/freeCodeCamp/freeCodeCamp/pull/59291/commits/0834eb24d3ef5a0ddb34744408734849331a6f11). Now only vfs is missing.

Reply:
The PR submitting vfs to cdnjs: https://github.com/cdnjs/packages/pull/1972